Detectors/MUON/MCH/Calibration/src/pedestals-decoding-workflow.cxx

```diff
@@ -78,18 +78,14 @@ static void patchPage(gsl::span<const std::byte> rdhBuffer, bool verbose)
 {
   static int mNrdhs = 0;
   auto& rdhAny = *reinterpret_cast<RDH*>(const_cast<std::byte*>(&(rdhBuffer[0])));
-  mNrdhs++;
 
   auto cruId = o2::raw::RDHUtils::getCRUID(rdhAny) & 0xFF;
   auto flags = o2::raw::RDHUtils::getCRUID(rdhAny) & 0xFF00;
   auto endpoint = o2::raw::RDHUtils::getEndPointID(rdhAny);
-  uint32_t feeId = cruId * 2 + endpoint + flags;
+  auto oldFeeId = o2::raw::RDHUtils::getFEEID(rdhAny);
```

Comment:
do we still need to patch the rdh ? I thought the decoder should now deduce automatically the ul version ? what is missing to drop the patchPage completely ?

Reply:
@aphecetche you are right, we can now remove the patching of the RDH. I will update the code.